docs/how-to/crafting/configure-package-information.rst

```diff
@@ -86,6 +91,11 @@ When crafting a snap, fill these keys as follows:
         case where your snap needs higher levels of system access, like a traditional
         unsandboxed package, you can :ref:`enable classic confinement
         <how-to-enable-classic-confinement>`.
+    * - ``common-id``
+      - Set a common id for your app(s). This common id is a reverse dns id used as
+        an appstream id of that app. This id useful in appstream centric stores. Please
+        look into `the documentation
+        <https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic>`_ for more info.
```

Comment:
```suggestion
```

I appreciate the detail, but this is a list of the _minimal_ keys needed for the average snap. It maps to the placeholder keys in an initialized project file.

If we want to make this standard, we'd have to add it to the template.

Reply:
This was suggested by @mr-cal [here](https://github.com/canonical/snapcraft/pull/5610#pullrequestreview-3051519026)

Reply:
If it's having that kind of impact on UX, then it sounds like it's larger than docs. If the result of `snapcraft init` can't pass a default linter, shouldn't we add it to the template?

Reply:
@soumyaDghosh, after discussing `common-id` with @medubelko and @lengau, we decided that we aren't ready to introduce `common-id` as a linter warning. The primary reason is that we don't define a common-id in the init template and it's not well-known enough for us to be comfortable linting for it.

We may decide to add a linter for it in the future, but in the interim, do you mind removing it from this PR? Sorry for the back and forth on this.

Reply:
Thanks a lot for the clarification @mr-cal. I will add some helper functions that I needed to get `common-id` from a `SnapMetadata`. I will write down those helpers for a future references and remove every fields related to `common-id`.